fusor-ember-cli/app/serializers/foreman-task.js

```diff
@@ -1,6 +1,7 @@
 import DS from 'ember-data';
 
-export default DS.RESTSerializer.extend({
+export default DS.ActiveModelSerializer.extend({
+  isNewSerializerAPI: true,
```

Comment:
I was getting errors if I didn't assign `isNewSerializerAPI: true`. Not sure what caused this change.